plugins/nginx_ingress.yaml

```
@@ -44,6 +44,13 @@ parameters:
       - beginning
       - end
     default: end
+  - name: source
+    label: Log source
+    description: Use this field to specify where your logs are coming from. When choosing the 'Kubernetes' options, the agent reads logs from /var/log/containers based on the Pod and Container specified below.
```

```suggestion
    description: Use this field to specify where your logs are coming from. When choosing the 'Kubernetes' options, the agent reads logs from /var/log/containers based on the Pod and Container specified.
```
The order of these show up within the OIQ UI in order so the parameters would be theoretically "above" this parameter.